packages/salesforcedx-vscode-apex/package.json

```diff
@@ -49,9 +52,89 @@
     "test": "node ./node_modules/vscode/bin/test",
     "test:unit": "node ./node_modules/vscode/bin/test"
   },
-  "activationEvents": ["workspaceContains:sfdx-project.json"],
+  "activationEvents": [
+    "workspaceContains:sfdx-project.json",
+    "onCommand:sfdx.force.test.view.showError",
+    "onView:sfdx.force.test.view"
+  ],
   "main": "./out/src",
   "contributes": {
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "test",
+          "title": "Test Viewer",
+          "icon": "media/dep.svg"
+        }
+      ]
+    },
+    "views": {
+      "test": [
+        {
+          "id": "sfdx.force.test.view",
+          "name": "Apex Tests"
+        }
+      ]
+    },
+    "menus": {
+      "view/title": [
+        {
+          "command": "sfdx.force.test.view.run",
+          "when": "view == sfdx.force.test.view",
```

Comment:
We need to come up with an analysis of whether we can support multiple test panels since that will inform us on how to continue.